golos.publication/golos.publication.cpp

```diff
@@ -281,6 +290,66 @@ void publication::create_message(
     });
 }
 
+void publication::addpermlink(structures::mssgid msg, structures::mssgid parent, uint16_t level, uint32_t childcount) {
+    require_auth(_self);
+    uint64_t parent_pk = 0;
+    if (parent.author) {
+        eosio::check(parent.permlink.size() > 0, "Parent permlink must not be empty");
```

Comment:
In case of `addpermlinks` action this message doesn't give information about failed permlink. It's will be good to add some information to understand which on which permlink fails.